README.md

```diff
@@ -235,7 +237,7 @@ to use further services.
 
 * ``python-openstackclient`` must be installed
 * Heat, the OpenStack orchestration service,  must be usable on the cloud environment
-* a ``clouds.yml`` and ``secure.yml`` must be created (https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#clouds-yaml)
+* a ``clouds.yml`` and ``secure.yml`` must be created (https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#clouds-yaml) or alternatively (not recommended) the old ``OS_`` environment setting style be used (via sourcing an approipriate ``openrc`` file).
```

Comment:
appropriate

Reply:
Sorry, I should have spotted the TOC ... Fixed.
Spell fixes for Ceph and appropriate.